src/test/java/hw6/exercise1/SoftAssertWebElementsTest.java

```diff
@@ -0,0 +1,77 @@
+package hw6.exercise1;
+
+import hw6.BaseTest;
+import hw6.pages.JdiTestingHomePage;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.Arrays;
+import java.util.List;
+
+
+public class SoftAssertWebElementsTest extends BaseTest {
```

Comment:
Why do you need this test in the current hw?

Reply:
Deleted